doc/manual/manual.tex

```diff
@@ -7101,6 +7101,73 @@ \subsubsection{Simple convection in a spherical 3d shell}
 with a more elaborate material model we should be able to get even closer to
 reality.
 
+\subsubsection{Postprocessing spherical 3D convection}
+\label{sec:cookbooks-shell_3d_postprocess.prm}
+\textit{This section was contributed by Jacqueline Austermann, Ian Rose and Shangxin Liu}
+
+
+There are several postprocessors that can be used to turn the the velocity 
+and pressure solution into quantities that can be compared to surface 
+observations. In this cookbook we introduce two postprocessors: dynamic 
+topography and the geoid. We initialize the model with a harmonic 
+perturbation of degree 4 and order 2 and calculate the
+instantaneous solution. Analogous to the previous setup we use a spherical
+shell geometry model and a simple material model. 
+
+The relevant section in the input file that determines the postprocessed
+output is as follows:
+
+\lstinputlisting[language=prmfile]{cookbooks/shell_3d_postprocess/shell_3d_postprocess.part.prm.out}
+
+This initial condition results in distinct flow cells that cause local up- and
+downwellings (Figure~\ref{fig:pp}). This flow deflects the top and bottom boundaries
+of the mantle away from their reference height, a process known as dynamic topography.
+The deflection of the surfaces and density perturbations within the mantle also 
+cause a perturbation in the gravitational field of the planet relative to the
+hydrostatic equilibrium ellipsoid. 
+
+\paragraph{Dynamic topography at the surface and core mantle boundary.}
+Dynamic topography is calculated at the surface and bottom of the domain 
+through a stress balancing approach where we assume that the radial stress at 
+the surface is balanced by excess (or deficit) topography. We use the 
+consistent boundary flux (CBF) method to calculate the radial stress 
+at the surface \cite{ZGH93}. For the 
+bottom surface we define positive values as up (out)
+and negative values are in (down), analogous to the deformation of the upper 
```

Comment:
@gassmoeller : Hm, I think she means "out of the core" and "into the core", which corresponds to "up" and "down".

Or did you mean to say that symmetry suggests that if you use "up (out)" in the first line, you should use "down (in)" in the second?